src/web_interface/templates/home.html

```diff
@@ -48,90 +48,63 @@ <h4 class="mb-3">General Facts</h4>
-                            </tr>
-                        {% endfor %}
-                    </tbody>
-                </table>
+            <div class="p-2 mt-3" style="width: 705px;">
+                <h4 class="mb-3">Latest Firmware Submissions</h4>
+                <ul class="list-group">
+                    {% for firmware in latest_firmware_submissions %}
+                        {{ firmware | firmware_detail_tabular_field | safe }}
+                    {% endfor %}
+                </ul>
             </div>
     	{% endif %}
 
     </div>
 
     <div class="row justify-content-center">
     	{% if (latest_comments | length) > 0 %}
-            <div class="p-2">
+            <div class="p-2 mt-3" style="width: 705px;">
                 <h4 class="mb-3">Latest Comments</h4>
-                <table class="table table-sm table-hover" style="width: 705px;">
-                    <thead class="thead-light">
-                        <tr>
-                            <th class="align-middle text-center" style="padding:5px;">Submission Time</th>
-                            <th class="align-middle text-center" style="padding:5px;">Author</th>
-                            <th class="align-middle text-center" style="padding:5px;">Comment</th>
-                        </tr>
-                    </thead>
-                    <tbody>
-                        {% for comment in latest_comments %}
-                            <tr>
-                                <td style="font-size: 14px"> {{ comment.time | int | nice_unix_time }} </td>
-                                <td style="font-size: 14px"> {{ comment.author }} </td>
-                                <td class="clickable" onclick="location.href='/analysis/{{ comment.uid }}'">
-                                    <span style="font-family: monospace; color: #000; font-size: 14px">
-                                        {{ comment.comment | urlize }}
+                <ul class="list-group">
+                    {% for comment in latest_comments %}
+                        <li class="list-group-item p-2 clickable li-hover" onclick="location.href='/analysis/{{ comment.uid }}'">
+                            <a href="/analysis/{{ comment.uid }}">
+                                <div class="p-1" style="width: 100%; height: 100%; font-size: 14px;">
+                                    <span style="font-family: monospace; color: #515151; font-size: 14px; word-break: break-word;">
+                                        {{ comment.comment | safe }}
                                     </span>
-                                </td>
-                            </tr>
-                        {% endfor %}
-                    </tbody>
-                </table>
+                                    <br>
+                                    <span style="font-family: 'Courier New', Courier, monospace; font-size: 10px; color: #6a737d">
```

Comment:
`class="text-muted"` for timestamp and name simply in black

Reply:
done

Reply:
also added class "footnote" for the rest of the styling to reduce in-line style entries